src/ResponseValidator.ts

```diff
@@ -171,9 +171,6 @@ export class ResponseValidator {
             logger.throw(new Error("Expected code in response"));
         }
 
-        if (!state.code_verifier && response.code) {
-            logger.throw(new Error("Unexpected code in response"));
```

Comment:
here I was not sure what to do here I did not see a reason to save in state the pkcedisabled option,
instead if code verifier does not exist on client I do not check it. I can say for sure that a pkce enabled initiated login will not succeed at server side if you don't send the code, so I think this is safe to remove let me know if you think I'm wring.

Reply:
i expect in your case `state.code_verifier=false` and in the response no code. The if should still working...

Reply:
are you sure? what happens is code_verifier = false, but code does come back that's the whole point I can exchange code for a token without pkce.